src/Plane/PlaneFileGlue.cpp

```diff
@@ -208,3 +210,21 @@ PlaneGlue::WriteFile(const Plane &plane, Path path)
   buffered.Flush();
   file.Commit();
 }
+
+StaticString<42>
+PlaneGlue::GetFileName(const Plane &plane)
+{
+  StaticString<42> filename(plane.registration);
+  filename += _T(".xcp");
+  return(filename);
+}
+
+// Simplified interface to write plane file including file extension handling
+void
+PlaneGlue::WriteFile(const Plane &plane)
+{
+  StaticString<42> filename( GetFileName(plane) );
+  const auto path = LocalPath( filename );
```

Comment:
Simpler and more efficient:
```
  const auto path = LocalPath(GetFileName(plane));
```
This allows the compiler to generate code to free the stack space occupied by `GetFileName()` immediately after returning from `LocalPath()`.

Reply:
Sure, will be changed in PR for plane masses